pkg/datapath/connector/veth.go

```diff
@@ -92,8 +93,7 @@ func SetupVethWithNames(lxcIfName, peerIfName string, mtu, groIPv6MaxSize, gsoIP
 	// Disable reverse path filter on the host side veth peer to allow
 	// container addresses to be used as source address when the linux
 	// stack performs routing.
-	err = DisableRpFilter(lxcIfName)
-	if err != nil {
+	if err := sysctl.Disable(fmt.Sprintf("net.ipv4.conf.%s.rp_filter", lxcIfName)); err != nil {
```

Comment:
Why was this change required? Ok to keep it if you think this looks more straightforward, just wondering.

Reply:
I don't think it is. Seems like a half attempt at removing the exported function, but its still in use :smile:. I can clean that up.